bambam.py

```diff
@@ -224,18 +228,30 @@ def print_letter(self, char):
         textpos.centery = h
         self.screen.blit(text, textpos)
 
-    def glob_dir(self, path, pattern):
-        files = glob.glob(os.path.join(path, pattern))
-        dirs = [x for x in glob.glob(os.path.join(path, '*')) if os.path.isdir(x)]
-        for subdir in dirs:
-            files.extend(self.glob_dir(subdir, pattern))
+    def glob_dir(self, path, extensions):
+        files = []
+        for fn in os.listdir(path):
+            fn = os.path.join(path, fn)
+            if os.path.islink(fn):
```

Comment:
I think you wanted `isdir` here which will return True for both regular dirs and symlinks to dirs, but not symlinks to files?